syft/frameworks/torch/tensors/interpreters/pointer.py

```diff
@@ -309,3 +309,12 @@ def attr(self, attr_name):
         ).wrap()
         self.__setattr__(attr_name, attr_ptr)
         return attr_ptr
+
+    def share(self, *args, **kwargs):
```

Comment:
What does this return?

Reply:
Allows you to send an order to share remotely: useful for privacy when local worker shoudl'nt be trusted

Reply:
* on the way shares are done